index.html

```diff
@@ -21,27 +21,220 @@
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="grid-items">
+                    <div class="grid-card">
+                        <div class="post-card">
+                            <h4>Multi-Post Stories Gain+Glory</h4>
+                        </div>
+                        <div class="post-list">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="grid-items">
+                    <div class="grid-card">
+                        <div class="post-card">
+                            <h4>Multi-Post Stories Gain+Glory</h4>
+                        </div>
+                        <div class="post-list">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="grid-items">
+                    <div class="grid-card">
+                        <div class="post-card">
+                            <h4>Multi-Post Stories Gain+Glory</h4>
+                        </div>
+                        <div class="post-list">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                
+                
+                
+                
+                
+            </div>
+        </section>
+        <section>
+            <div class="flex-container">
+                <div id="about-title">
+                    <div id="about-box" class="font-normal">
+                        <p>About me</p>
```

Comment:
- [ ] I kindly suggest that you make use of an `hx` heading tag to display the `About me` text. It is important to make use of the semantic heading tag to display the major content of a particular section of your website :+1: